system/Cookie/Cookie.php

```diff
@@ -630,7 +626,7 @@ public function toHeaderString(): string
     /**
      * {@inheritDoc}
      */
-    public function __toString()
+    public function __toString(): string
```

Comment:
Is this a breaking change?

Reply:
No, `__toString` is special, it keep working even child doesn't has return type